spec/features/users_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :feature do
+  before do
+    @user1 = User.create!(name: 'Tom', posts_counter: 0)
+    @user2 = User.create!(name: 'Toro', posts_counter: 0, bio: 'developer')
+    @user3 = User.create!(name: 'Ted', posts_counter: 0)
+
+    visit users_path
+  end
+
+  describe 'index page' do
+    it 'displays usernames of all other users' do
+      expect(page).to have_content('Tom')
+      expect(page).to have_content('Toro')
+      expect(page).to have_content('Ted')
+    end
+
+    it 'displays profile picture for each user' do
+      expect(page).to have_css('div.user-photo', count: 5)
+    end
```

Comment:
- [ ] It would be better if we fix the error in the test as seen below
![Screenshot from 2023-07-05 21-55-02](https://github.com/Odongo006/Rails_blog_App/assets/46388113/37c71110-351d-4005-849d-35cf28d594da)

Reply:
- [ ] @Odongo006 as the previous reviewer noted the `display profile picture for each user` test is failing, you created 3 sample users but you are testing to see if it displays 5 sections which it won't find making the test fail
![image](https://github.com/Odongo006/Rails_blog_App/assets/61408860/3c6eedf6-a7fd-48b3-aded-934f6d019d7f)